### PR TITLE
Add russian translation

### PR DESCRIPTION
**Pull request by enl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/.idea
```
Comment (craue) [inline .gitignore]: Please move it down one line to be in alphabetic order.

File: Resources/translations/CraueConfigBundle.ru.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+no_settings_defined: Никаких настроек пока не создано.
+modify_settings: Применить
```
Comment (craue) [inline Resources/translations/CraueConfigBundle.ru.yml]: Should be lowercased as it's not a sentence.

**Comment by craue** [comment]
Would you rewrite the history to have only two nice commits (one for .gitignore, one for the translation)? If not, I can do this as well.

**Comment by enl** [comment]
@craue, frankly speaking, I have no idea how to do this.

**Comment by craue** [comment]
Done. :smirk:
Thank you, @enl.

**Pull request closed without merging**